cmd/eksctl-anywhere/cmd/get.go

```diff
@@ -56,9 +57,9 @@ func getResources(ctx context.Context, resourceType string, output string, args
 		return fmt.Errorf("kubectl execution failure: \n%v", err)
 	}
 	if len(stdOut.Bytes()) == 0 {
-		fmt.Printf("No resources found in %v namespace", constants.EksaPackagesName)
+		fmt.Printf("No resources found in %v namespace\n", constants.EksaPackagesName)
```

Comment:
Shouldn't this make use of logger instead of a printf?

Reply:
Actually this needs to be printed to the user.